Navaids/NDB_All.txt

```diff
@@ -50,7 +50,7 @@ OY  332.000 N054.41.33.530 W006.05.07.300 ;Belfast Aldergrove
 PIK 355.000 N055.30.21.920 W004.34.38.110 ;Prestwick
 RWY 359.000 N054.04.51.900 W004.37.22.400 ;Ronaldsway/Isle of Man
 SAY 431.000 N058.12.55.720 W006.19.44.680 ;Stornoway
-SBH 351.000 N059.52.56.560 W001.17.41.390 ;Sumburgh
+SBH 351.000 N059.52.43.340 W001.17.11.490 ;Sumburgh
```

Comment:
Coordinates weren't quite correct in the issue. In the 2203 eAIP on the EGPB AD pages, I find: 595256.56N 0011741.40W
```suggestion
SBH 351.000 N059.52.56.560 W001.17.41.400 ;Sumburgh
```